src/app/modules/pictureapp-explore/components/edit-userProfile/components/edit-user-photos/edit-user-photos.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="example-container">
+  <form [formGroup]="editPhotoForm" (submit)="submitChanges()">
+    <mat-card class="edit-user-photos-card" [ngClass]="{'edit-user-photos-card-current': isEdit, 'edit-user-photos-card-removing': isInRemoving}"> 
+          <mat-card-header>
+              <div class="example-container">
+                <mat-card-title>
+                  <label *ngIf="!isEdit">{{photo.title}}</label>
+                  <mat-form-field *ngIf="isEdit" class="edit-user-photos-card-inputs">
+                    <input
+                      matInput
+                      placeholder="edit title"
+                      formControlName="photoTitle"
+                    />
+                  </mat-form-field>
+                </mat-card-title>
+                <mat-card-subtitle>
+                  <label *ngIf="!isEdit">{{photo.subtitle}}</label>
+                  <mat-form-field *ngIf="isEdit" class="edit-user-photos-card-inputs">
+                      <input
+                        matInput
+                        placeholder="edit subtitle"
+                        formControlName="photoSubtitle"
+                      />
+                    </mat-form-field>                  
+                </mat-card-subtitle>
+              </div>
+            </mat-card-header>   
+            <img mat-card-image src="{{ photo.url }}" alt="Photo of a Shiba Inu" />
```

Comment:
I think we should load data for 'alt' tag dynamically, from data store or simply remove this tag here.

Reply:
ok removed that.